bindings/go/evmc/evmc.go

```diff
@@ -120,7 +126,7 @@ func (err Error) Error() string {
 		return fmt.Sprintf("evmc: unknown internal error (%d)", int32(code))
```

Comment:
Maybe remove "unknown" from here as we do not predefine any internal errors at the moment.

Reply:
Perhaps "specific internal error" ? "user-define internal error" ?

Reply:
I'd go with "internal error" only.

Reply:
Then the other `internal error` should also have a value IIRC.